package.json

```diff
@@ -15,13 +15,14 @@
   "dependencies": {
     "@serverless/core": "^1.1.2",
     "@serverless/tencent-apigateway": "^2.0.0",
-    "@serverless/tencent-scf": "^2.0.2",
-    "aws-serverless-express": "^3.3.6",
+    "@serverless/tencent-scf": "^2.0.5",
     "ext": "^1.4.0",
     "fs-extra": "^8.1.0",
     "prettier": "^1.19.1",
+    "tencent-serverless-nodejs": "^1.0.0",
     "terser": "^4.4.2",
     "type": "^2.0.0",
+    "type-is": "^1.6.18",
```

Comment:
I think this dependency is no longer needed (?)

Reply:
Yes, I had removed it.